Snail/Unique.swift

```diff
@@ -17,4 +17,9 @@ public class Unique<T: Equatable>: Variable<T> {
             subject.on(.next(newValue))
         }
     }
+
+    public override init(_ value: T) {
+        super.init(value)
+        subject.on(.next(value))
```

Comment:
@0x107 can you explain the benefit of this? Both `Variable` and `Unique` should behave the same way, and iirc the existing behavior is intentional. I'm open to the change, just curious the use case for it.

Reply:
@wesbillman I found out, that `Variable` and `Unique` behaves different on subscription, when they still have their initial value. `Variable` notifies the subscriber and sends the value but `Unique` does not.

When i subscribe to a `Unique` property of my view-model, which initial value does not have been altered yet,  i don't get the value.

With this constructor I want to achieve the same behavior of `Variable` and `Unique`.

Reply:
@0x107 thanks!

Just to make sure I understand the motivation. This is essentially making `Unique` behave the same way `Variable` does? In that you get a notification of the initial value:
https://github.com/UrbanCompass/Snail/pull/92

Assuming that's true, I'm very happy with this change :)